README.md

```diff
@@ -696,45 +700,45 @@ ones discussed in that analysis, but they give a sense of how ownership can be d
 
 ### Stages of a startup
 
-Understanding stock and equity value in a startup requires understanding the stages of a
-startup’s growth and how that affects who owns the stock.
-
-- The stage of a startup is largely reflected in how much funding it has raised.
-  Very roughly,
-  [typical levels](http://blog.eladgil.com/2011/03/how-funding-rounds-differ-seed-series.html)
-  are:
-  - **Bootstrapped** (No funding): Founders are figuring out what to build, or they’re starting
-    to build with their own time and resources.
-  - **Series Seed** ($250K to $2 million): Figuring out the product and market.
-  - **Series A** ($2 to $15 million): Scaling the product and making the business model work.
-  - **Series B** (tens of millions): Scaling the business.
-  - **Series C, D, E, etc.**
-    (tens to hundreds of millions): Continued scaling of the business.
+Understanding the value of stock and equity in a startup requires a grasp of the stages of
```

Comment:
OK for now, but if we rewrite this there's more to be put here on motivation. Should also mention increased equity in exchange for risk and risk decreasing as a startup matures.